webform_import.module

```diff
@@ -138,11 +138,22 @@ function webform_import_form($node, $form_state) {
     array('data' => t('Field Names') . '&nbsp;<div class="move-right"> ' . $name_button . ' </div>'),
     array('data' => t('Field Form Keys') . '&nbsp;<div class="move-right"> ' . $key_button . ' </div>'),
   );
+
+  // Standard webform components
```

Comment:
This inline comment can probably be removed. Or, we should end it with a period. Or Ideally, it should be rewritten as a sentence. It's also fine :)